src/components/navbar.tsx

```diff
@@ -28,12 +28,11 @@ function Navbar() {
           document.querySelector(NAV_SELECTOR)?.classList.add('lg:hidden');
         }}
       >
-        <ul className="flex w-full items-center justify-center gap-4">
+        <ul className="hidden w-full items-center justify-center gap-4 lg:flex">
```

Comment:
C'est pas plutôt md ou sm ?

```suggestion
        <ul className="hidden w-full items-center justify-center gap-4 md:flex">
```